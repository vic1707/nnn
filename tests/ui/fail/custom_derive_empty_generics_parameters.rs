use nnn::nnn;

#[nnn(nnn_derive(Into<>, From<>))]
struct Float(f64);

fn main() {}
