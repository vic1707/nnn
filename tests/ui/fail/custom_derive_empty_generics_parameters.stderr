error: Please provide generics arguments, or omit the '<>' for the default derive.
 --> tests/ui/fail/custom_derive_empty_generics_parameters.rs:3:22
  |
3 | #[nnn(nnn_derive(Into<>, From<>))]
  |                      ^^
